frontend/src/components/PlayComponents/BetCard.js

```diff
@@ -50,4 +80,20 @@ const BetCardStyled = styled.div`
   grid-gap: var(--size-s);
   grid-auto-rows: min-content;
   padding: var(--size-s);
+
+  p {
+    margin: 0;
+  }
+  form {
+    display: flex;
+    justify-content: flex-start;
+  }
+  label {
+    display: grid;
+    grid-gap: var(--size-s);
+    grid-template-columns: 1fr 1fr;
+  }
+  input {
+    max-width: 5em;
```

Comment:
Why is this related to the font-size? Maybe make it a fraction of the available space?